rules/configuration.smk

```diff
@@ -417,11 +424,12 @@ def any_given(*args):
         elif column_name is 'sample':
             elements.extend(samples.index)
 
+    elements = [element for element in elements if isinstance(element, str)]
```

Comment:
samples is loaded with dtype = 'str', so this shouldn't be needed (?)
can be left in tho

Reply:
So the PR's got a bit mixed up, but I added the option to add a control to your samples.tsv. This field can also be left empty, so that's why I check if it's a string, since nan isnt.

Reply:
sample assembly control
GSM123 hg38 GSM321
GSM432 hg38